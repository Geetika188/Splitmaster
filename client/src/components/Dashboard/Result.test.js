import React from 'react';
import { mount, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import Result from './Result';

configure({ adapter: new Adapter() });

it('should render correctly', () => {
  const component = mount(<Result/>);
  expect(component).toMatchSnapshot();
  component.unmount();
});
