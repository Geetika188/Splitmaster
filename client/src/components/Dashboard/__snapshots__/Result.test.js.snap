// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Result>
  <div>
    <ul>
      <li
        className="friendlist"
      >
        <i
          class="fas fa-user"
        />
      </li>
    </ul>
  </div>
</Result>
`;
